src/main.rs

```diff
@@ -120,6 +120,13 @@ struct Args {
     /// (long_about: "Prompt (if using llm for translation)")
     #[arg(long, default_value = "")]
     llm_prompt: String,
+
+    /// ggml model path
+    /// (default: "ggml-tiny.bin")
+    /// (example: "ggml-tiny.bin", ggml-small.bin")
+    /// (long_about: "Path to the ggml model")
+    #[arg(long, default_value = "ggml-tiny.bin")]
+    ggml_model_path: String,
```

Comment:
Could you put this closer to the `subtitle_source` arg? Because they are related args.

Reply:
of course

Reply:
Done